ecl/hql/hqlexpr.cpp

```diff
@@ -10810,6 +10816,8 @@ IHqlExpression *createDictionary(node_operator op, HqlExprArray & parms)
         }
     case no_nofold:
     case no_nohoist:
+    case no_forcegraph:
```

Comment:
If it was moved, it was so it was grouped with the operators that logically the same - i.e. don't generate activities, but try and force some change in the graph.  This example is a case of adding it because it probably should have been in the case statement.  (no_forcegraph is almost never used...)